spec/helpers/thirdParty.js

```diff
@@ -73,6 +97,245 @@ describe('third party handlebars-helpers', function() {
                     },
                 ], done);
             });
+
+            it('should evaluate inner expression when haystack contains needle', function(done) {
+                runTestCases([
+                    {
+                        input: '{{#contains haystack needle}}FOUND{{/contains}}',
+                        output: 'FOUND',
+                        context: {
+                            needle: "hello",
+                            haystack: "hello world"
+                        },
+                    },
+                    // {
```

Comment:
Is it a temporary block of commented code that should be removed, or there is some special sense in it?

Reply:
These test are failing bcoz even though the helper says it supports object it doesn't